scripts/ccpp_track_variables.py

```diff
@@ -170,20 +166,29 @@ def create_var_graph(suite, var, config, metapath, run_env):
 
     return (success,var_graph)
 
-def main():
+def track_variables(sdf,metadata_path,config,variable,debug):
     """Main routine that traverses a CCPP suite and outputs the list of schemes that use given
-       variable, broken down by group"""
+       variable, broken down by group
 
-    args = parse_arguments()
+    Args:
+        sdf           (str) : The full path of the suite definition file to parse
+        metadata_path (str) : path to CCPP scheme metadata files
+        config        (str) : path to CCPP prebuild configuration file
+        variable      (str) : variable to track through CCPP suite
+        debug        (bool) : Enable extra output for debugging
+
+    Returns:
+        None
+"""
 
-    logger = setup_logging(args.debug)
+    logger = setup_logging(debug)
 
     #Use new capgen class CCPPFrameworkEnv
     run_env = CCPPFrameworkEnv(logger, host_files="", scheme_files="", suites="")
 
-    suite = parse_suite(args.sdf,run_env)
+    suite = parse_suite(sdf,run_env)
 
-    (success, config) = import_config(args.config, None)
+    (success, config) = import_config(config, None)
```

Comment:
You also do not need the tuple to unpack the variables. Same questions as above.

Reply:
But it doesn't hurt, or? If so, then does it matter?